direct/src/showbase/Loader.py

```diff
@@ -976,6 +976,10 @@ def loadSound(self, manager, soundPath, positional = False,
                 sound = manager.getSound(soundPath, positional)
                 result.append(sound)
 
+                if str(sound).startswith('NullAudioSound') and not okMissing:
```

Comment:
This is not a clean way to check for this.  You can compare to `manager.getNullSound()` instead.